### PR TITLE
Adding addressbook select on each contact

### PR DESCRIPTION
**Pull request by DeepDiver1975**
@Henni @jancborchardt @irgendwie @nickvergessen @LukasReschke

### CODE REVIEW

File: js/components/contactDetails/contactDetails_controller.js [hunk 1]
```diff
@@ -8,6 +8,17 @@ app.controller('contactdetailsCtrl', ['ContactService', 'vCardPropertiesService'
...
 	ctrl.fieldDefinitions = vCardPropertiesService.fieldDefinitions;
+	$scope.addressBooks = [];
+
+	AddressBookService.getAll().then(function(addressBooks) {
+		//$scope.addressBooks = addressBooks.map(function (element) {
+		//	return {
+		//		id: element.displayName,
+		//		name: element.displayName
+		//	};
+		//});
```
Comment (Henni) [inline js/components/contactDetails/contactDetails_controller.js]: Can we remove this comment?

**Comment by nickvergessen** [comment]
Everytime I select a different addressbook, the entry is duplicated for me, at least when I change it more then once.

**Comment by DeepDiver1975** [comment]
> Everytime I select a different addressbook, the entry is duplicated for me, at least when I change it more then once.

can you please rebase? #65 was merged meanwhile which addresses this afaik

**Comment by nickvergessen** [comment]
> can you please rebase? #65 was merged meanwhile which addresses this afaik

Right, works no more duplicates

However, still one problem remaining:
1. Select a contact
2. Change addressbook
3. Content is refreshed automatically
4. Select same contact
5. Addressbook dropdown shows the old addressbook

**Comment by DeepDiver1975** [comment]
@nickvergessen @Henni please test - thx

**Comment by irgendwie** [comment]
LGTM :+1:

**Comment by Henni** [comment]
This broke creating contacts
edit: and group switching

**Pull request merged**